websocket_tls/nbioclient_ordered_messages/client.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/url"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/lesismal/nbio/nbhttp"
+	"github.com/lesismal/nbio/nbhttp/websocket"
+
+	"github.com/lesismal/llib/std/crypto/tls"
+)
+
+var (
+	clients           = flag.Int("clients", 1, "number of clients")
+	connectedClients  chan *websocket.Conn
+)
+
+func newUpgrader() *websocket.Upgrader {
+	u := websocket.NewUpgrader()
+	u.OnMessage(func(c *websocket.Conn, messageType websocket.MessageType, data []byte) {
+		fmt.Printf("received: %s\n",string(data))
+		connectedClients <- c
```

Comment:
3. when the first echo msg sent by the server is received, you put the client into the chan again, and here's nobody get the client from the chan, and the chan's queue is full.
4. when the second echo msg sent by the server is received and you put the client into the chan again, it's blocked because the chan's queueue is full.

Reply:
ah yeah, missed that. Sorry and, thank you